test/unit/presenters/publishing_api/html_attachment_presenter_test.rb

```diff
@@ -133,13 +132,8 @@ def present(record)
       ],
     )
 
-    expected = [
-      { title: "Link 1", href: "https://www.example.com/1" },
-      { title: "Link 2", href: "https://www.example.com/2" },
-    ]
-
     html_attachment = HtmlAttachment.last
 
-    assert_equal present(html_attachment).content[:details][:brexit_no_deal_notice], expected
+    assert_nil present(html_attachment).content[:details][:brexit_no_deal_notice]
```

Comment:
I'll remove these bits in the db migration work 👍

Reply:
Great, thank you!